VialerTests/Middleware/MiddlewareTests.m

```diff
@@ -57,8 +57,15 @@ - (void)testMiddlewareRequestOperationManagerPropertyCreation {
 }
 
 - (void)testReachabilityManagerCreation {
-    //The Assert calls the getter which will lazy load de registry which succeeds the test.
-    XCTAssert([self.middleware.reachabilityManager isKindOfClass:[ReachabilityManager class]]);
+    //The Assert calls the getter which will lazy load the registry which succeeds the test.
+    //XCTAssert([self.middleware.reachabilityManager isKindOfClass:[ReachabilityManager class]]); //old code, reachabilityManager is not anywhere
+    
+    XCTAssert([self.middleware.reachabilityHelper  isKindOfClass:[ReachabilityHelper class]]);          //this test fails
```

Comment:
The Middleware class doesn't have a ReachabilityHelper interface
`failed: throwing "-[Middleware reachabilityHelper]: unrecognized selector sent to instance 0x60400067c3c0`
The Middleware has a Reachability property. so it doesn't detect the reachabilityHelper property.

So the test will actually be:
`XCTAssert([self.middleware.reachability isKindOfClass:[Reachability class]]);  `